### ISSUE TITLE
Accessibility feedback for Staging Review – Public Websites [Resources and Support]

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#15378]
# 🏆  Results of audit 🎉  

While the content is in the VA environment, we are limited on the audits we can run. I checked everything except the three screen readers, and it was **spotless!**

Once the work is in production, during the Full Accessibility & 508 Office Audit I'll rescan and check in the three screen readers. We don't have access to Safari+VoiceOver, Firefox+NVDA, and IE11+JAWS using the SOCKS proxy.

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Point of Contact

VSA: Jennifer
VSP: Trevor

## Details

This epic contains the accessibility feedback for the Staging Review for Public Websites [Resources and Support].

### Tested

1. http://learningcenter.web.demo.ci.cms.va.gov/node/8540/

1. http://learningcenter.web.demo.ci.cms.va.gov/taxonomy/term/282/

1. http://learningcenter.web.demo.ci.cms.va.gov/node/8222/

1. http://learningcenter.web.demo.ci.cms.va.gov/taxonomy/term/293/

### Unable to access

1. www.va.gov/resources/search/?query=claim+status

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

**Comment by 1Copenut** [comment]
VSP concurs! Things are right where they need to be and a plan is in place for testing once content is ready with URLs.
